code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -396,6 +396,29 @@
 		/obj/item/seeds
 	))
 
+/**
+  * # Food and Drink Cart
+  *
+  * Variant of the [Smart Fridge][/obj/machinery/smartfridge] that holds food and drinks in a mobile form
+  */
+/obj/machinery/smartfridge/foodcart
```

Comment:
You probably want to have an override here for `screwdriver_act()` and maybe `crowbar_act()` too, otherwise it's possible to deconstruct the food cart into smartfridge components:

https://user-images.githubusercontent.com/57483089/136048727-d6d1ea50-1d9e-4d9d-a315-56ca658b24d2.mp4

Same issue too if it's destroyed manually. The `/obj/structure/foodcart` version had a `deconstruct()` override, so copying that over to this one should fix it.